docs/development/switcheroo.md

```
@@ -0,0 +1,53 @@
+
+_(Having our own integration would've been better tbh)_
+
+## Service
+
+- **Interface:** `net.hadess.SwitcherooControl`
+
+---
+
+## Properties
+
+### `HasDualGpu`
```

markdownlint-cli2 (0.23.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)